src/lib/routes/util.ts

```diff
@@ -56,6 +56,8 @@ export const handleErrors: (
             return res.status(403).json(error).end();
         case 'MinimumOneEnvironmentError':
             return res.status(400).json(error).end();
+        case 'OperationDeniedError':
+            return res.status(403).json(error).end();
```

Comment:
We could group all the 403 cases together.

Reply:
Agreed. I'll fix

Reply:
I grouped statuses together in ascending order, so all 400, then 401, then 403, then 409.

Reply:
makes sense